scene/gui/tab_bar.cpp

```diff
@@ -1278,5 +1532,6 @@ void TabBar::_bind_methods() {
 }
 
 TabBar::TabBar() {
+	set_size(Size2(get_size().width, get_minimum_size().height));
```

Comment:
What is this line for?

Reply:
Control nodes start with a size of 40x40 at the start. However, `TabBar` with the default is shorter in height than that, so this line makes sure it isn't unnecessary taller than it should.